benchmarks.go

```diff
@@ -454,6 +455,15 @@ var benchmarkCases = []BenchmarkCase{
 		UnsafeStringUnmarshal: true,
 		TimeSupport:           TSNoSupport,
 		APIKind:               AKManual,
+	}, {
+		Name: "idr",
+		URL:  "github.com/chmike/ditp",
+		New:  idr.NewIDRSerializer,
+
+		BufferReuseMarshal: true,
+		TimeSupport:        TSFullTzOffset,
+		APIKind:            AKManual,
+		Notes:              []string{"low level IDR encoding demo with same nsec time encoding as benc"},
```

Comment:
Capitalize the note ("Low level...") and you can drop the "same nsec time encoding"